change/@ni-nimble-components-05fac024-430c-46fd-86e3-987b72a3b15a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Spent some time getting the spec run times to log with both main and the foundation update branch:

https://github.com/ni/nimble/pull/2460
https://github.com/ni/nimble/pull/2459

Saved the log output locally and ran the following to grab the number of tests taking more than 500ms to run and printing the timings out:

```
passing-foundation-1a.txt length: 140 [503,505,507,507,507,510,511,513,514,516,516,517,519,519,521,521,524,525,526,526,527,534,535,536,538,539,539,541,543,543,547,548,552,553,554,555,556,559,559,560,562,565,567,584,589,597,600,601,602,603,604,610,611,615,616,616,623,624,630,632,635,639,647,660,663,670,670,675,676,684,685,689,693,699,699,704,706,709,717,720,730,734,753,771,774,785,790,795,797,808,841,855,883,886,887,887,945,952,960,981,983,989,993,1009,1016,1018,1020,1031,1035,1044,1047,1052,1068,1143,1147,1169,1200,1203,1243,1264,1299,1312,1325,1337,1352,1377,1446,1473,1535,1586,1631,2132,2166,2175,2242,2445,2618,3230,3292,4125]
passing-foundation-1b.txt length: 121 [503,505,509,509,510,512,516,518,519,519,521,521,522,523,524,527,532,532,535,536,536,540,547,548,554,555,555,563,565,570,570,573,578,578,578,580,583,590,590,600,604,610,610,641,650,651,658,660,664,665,667,668,668,670,670,679,697,704,707,707,708,733,750,757,765,774,778,784,792,797,801,811,812,814,850,873,895,925,931,934,942,954,959,972,982,990,1003,1010,1018,1020,1095,1105,1115,1115,1123,1129,1163,1199,1202,1237,1245,1262,1288,1305,1311,1376,1398,1434,1566,1740,1796,1822,1998,2046,2192,2275,3253,3532,4021,4267,4620]
passing-main-1a.txt length: 76 [503,505,509,512,514,517,523,524,525,525,526,528,529,534,537,540,542,547,552,558,559,560,560,564,567,576,577,578,578,578,592,596,596,596,599,604,610,613,616,628,629,631,635,648,650,652,668,669,673,674,679,689,701,711,724,747,751,778,790,804,809,846,849,851,852,853,855,884,930,932,938,957,1063,1253,1512,2320]
passing-main-1b.txt length: 67 [505,505,514,518,520,521,521,521,526,543,547,550,558,558,559,571,571,579,581,582,594,596,607,613,615,621,621,631,638,655,655,667,670,679,679,685,687,687,693,702,715,725,730,744,770,777,803,809,849,849,874,874,906,912,921,943,944,954,964,996,1011,1022,1074,1088,1115,1204,1232]
```

Script used:

```ts
log('passing-foundation-1a.txt');
log('passing-foundation-1b.txt');
log('passing-main-1a.txt');
log('passing-main-1b.txt');

function log(filename: string) {
    const text = Deno.readTextFileSync(filename);
    const result = [...text.matchAll(/.*\((\d*)ms\)/gm)].map(v => Number(v[1])).sort((a,b)=> a-b).filter(x => x>500);
    console.log(filename, `length: ${result.length}`, JSON.stringify(result));
}
```

Reply:
Resolved by @m-akinc's investigation in https://github.com/ni/nimble/issues/2456